docs/01_nodeos/03_plugins/net_api_plugin/index.md

```diff
@@ -8,7 +8,7 @@ The `net_api_plugin` provides four RPC API endpoints:
 * connections
 * status
 
-See [Net API Reference Documentation](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/net_api_plugin/api-reference/index).
+See [Net API Reference Documentation](http://docs.eosnetwork.com/reference/mandel-plugins/net_api.html).
```

Comment:
I think we should get https working ASAP, probably even before this PR does it. Bad look to not have https.